### PR TITLE
[New] Add PrintArea@2.4.1 w/ git auto-update

### LINKED ISSUE #8826: [Request] Add jquery.printarea.js
**Library name:** jquery.printarea.js
**Git repository url:**
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
close #8826, cc @RitsC. using koala tool for minifing

Pull request for issue: #8826
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/RitsC/PrintArea
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 23
   - Count of stars: 202
   - Count of forks: 133
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery.printarea/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "Djuri Baars"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/RitsC/PrintArea.git",
+    "fileMap": [
+      {
+        "basePath": "demo",
+        "files": [
+          "jquery.PrintArea*"
```
Comment (sufuf3) [inline ajax/libs/jquery.printarea/package.json]: Ummm I think use `demo`as the basePath is a little weird.
What do you think?

@extend1994 @maruilian11 Could you give some suggestion about this issue?

Many thanks.

Comment (sashberd) [reply ajax/libs/jquery.printarea/package.json]: This is author build folder... I can change it to regular css and js and minifiy them. However, for you suggestion.
I found that pattern in autoupdate was wrong. Was updated. FYR

File: ajax/libs/jquery.printarea/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "jquery.printarea",
+  "filename": "jquery.PrintArea.min.js",
+  "version": "2.4.1",
+  "description": "Prints a specific area of the page. Using json settings, the printed area is either opened in a popup or in a hidden iframe. Minimal setup and easy to use. See the demo for examples and usage of the options.",
+  "keywords": [
+    "jquery",
+    "print",
+    "printing"
```
Comment (sufuf3) [inline ajax/libs/jquery.printarea/package.json]: Let's also add `PrintArea` as keyword.
Thank you.

File: ajax/libs/jquery.printarea/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RitsC/PrintArea.git"
+  },
+  "authors": [
+    "Chris Ritschard",
+    "Djuri Baars"
```
Comment (sufuf3) [inline ajax/libs/jquery.printarea/package.json]: The author is `Chris Ritschard`
https://github.com/RitsC/PrintArea/blob/master/PrintArea.jquery.json#L7
Djuri Baars is not the author.
Thanks

Comment (sashberd) [reply ajax/libs/jquery.printarea/package.json]: @sufuf3 Where do you see PrintArea.css?

Comment (sufuf3) [reply ajax/libs/jquery.printarea/package.json]: Here, https://github.com/RitsC/PrintArea/blob/2.4.1/demo/PrintArea.css

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

**Comment by sashberd** [comment]
@sufuf3 ping

File: ajax/libs/jquery.printarea/package.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RitsC/PrintArea.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/RitsC/PrintArea.git",
+    "fileMap": [
+      {
+        "basePath": "demo",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.printarea/package.json]: hmmm, using `demo` as `basePath` is wired ...

**Comment by sashberd** [comment]
@PeterDaveHello I used wget to re-add css file and updated lib name. FYR

**Comment by sashberd** [comment]
@PeterDaveHello As I said in previous PR with same issue  the problem is in operating system. I had tried to fix it via [http://shanelabs.com/blog/2012/06/27/getting-rid-of-m-newline-characters-in-notepad/](url) but you can see still no luck. The only solution is readd those problematic files from you side(Linux, Unix)

**Comment by PeterDaveHello** [comment]
@sashberd than it's not fixed, you can simply verify the issue by calculating the checksum of the raw file from both upstream and GitHub(the one you committed), then you'll know if it's fixed or not.

**Comment by sashberd** [comment]
@PeterDaveHello I just do last PR from my miniMac. I checked sum of .css file and it was the same as in the main stream version.I hope the PR`s from Mac will not do ^M problem in files.
FYR

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

**Comment by PeterDaveHello** [comment]
Maybe @sashberd can help open an issue to upstream to confirm the new versoin?

**Comment by sashberd** [comment]
@sufuf3 I do not see version 2.4.2.. Can you provide me a link to it?

**Comment by sufuf3** [comment]
@sashberd 
FYI https://github.com/RitsC/PrintArea/blob/master/CHANGELOG#L1

**Comment by sashberd** [comment]
@sufuf3 Would you like I will open Beginner issue to fetch v2.4.2 from https://github.com/RitsC/PrintArea/tree/47504ac619e929a4ca6cedc9a37e56062556269b?

**Comment by sufuf3** [comment]
No, just need your help to discuss adding the v2.4.2 git tag on the lib's repository with the author.
Would you please help do this?
Like https://github.com/googlemaps/v3-utility-library/issues/366
Many thanks.

**Pull request merged**